win32ss/printing/base/winspool/printers.c

```diff
@@ -448,10 +449,20 @@ EnumPrintersA(DWORD Flags, PSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cbB
     }
  
     /* Ref: https://stackoverflow.com/questions/41147180/why-enumprintersa-and-enumprintersw-request-the-same-amount-of-memory */
-    bReturnValue = EnumPrintersW(Flags, pwszName, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
-    HeapFree(hProcessHeap, 0, pwszName);
+    bResult = EnumPrintersW(Flags, pwszName, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
+    dwErrorCode = GetLastError();
 
-    TRACE("*pcReturned is '%d' and bReturnValue is '%d' and GetLastError is '%ld'.\n", *pcReturned, bReturnValue, GetLastError());
+    TRACE("*pcReturned is '%d' and bResult is '%d' and dwErrorCode is '%ld'.\n", *pcReturned, bResult, dwErrorCode);
+
+    if (pwszName)
```

Comment:
Now that we have the recurring `if (pointer) HeapFree(pointer)` pattern in this function, please move it into the `Cleanup` branch to stay consistent with the rest of the printing code.

Reply:
I concur. Done.